Sources/SmithyTestUtil/DummyIdentityResolver.swift

```diff
@@ -0,0 +1,17 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import struct SmithyIdentity.StaticAWSCredentialIdentityResolver
+
```

Comment:
The function below just returns a static identity resolver that returns dummy AWS credentials.
This is used for clients in request protocol tests instead of setting credentials somewhere on the host to be picked up by the default chain.